src/hotspot/os/windows/os_windows.cpp

```diff
@@ -3535,7 +3568,7 @@ char* os::pd_reserve_memory_special(size_t bytes, size_t alignment, size_t page_
   // the alignment is bound to the heap region size. So this reservation needs to
   // ensure that the requested alignment is met. When there is a requested address
   // this solves it self, since it must be properly aligned already.
-  if (addr == nullptr && alignment > page_size) {
+  if (addr == nullptr && alignment > GetLargePageMinimum()) {
```

Comment:
Is it necessary to verify that GetLargePageMinimum() > 0 for this path?

Reply:
yes

Reply:
Ok, I'm unresolving this comment until you update the code then.